examples/create_groups.ipynb

```diff
@@ -1,149 +1,599 @@
 {
- "metadata": {
-  "language_info": {
-   "codemirror_mode": {
-    "name": "ipython",
-    "version": 3
-   },
-   "file_extension": ".py",
-   "mimetype": "text/x-python",
-   "name": "python",
-   "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.9.0"
-  },
-  "orig_nbformat": 2,
-  "kernelspec": {
-   "name": "python390jvsc74a57bd0f83ddd6991e62697b8e1c78d35b449f256fb868317b9158045034ef8d7136941",
-   "display_name": "Python 3.9.0 64-bit ('cf-3.9.0': pyenv)"
-  }
- },
- "nbformat": 4,
- "nbformat_minor": 2,
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# TileDB Groups\n",
+    "\n",
+    "This notebook shows an example of how to use the `Group` and `GroupSchema` classes from TileDB-CF to create, inspect, open, read, and write data to arrays in a TileDB group.\n",
+    "\n",
+    "### Data\n",
+    "\n",
+    "The following arrays will be created that share some of their dimensions:\n",
+    "\n",
+    "* `D_xyt` is a dense array with dimensions (`Dx`,`Dy`,`Dt`)\n",
```

Comment:
I think slightly more versbose names might be easier to follow. Maybe something like `dense_xyt` instead of `D_xyt`?